web/page0/.gitignore

```diff
@@ -40,3 +40,6 @@ testem.log
 # System files
 .DS_Store
 Thumbs.db
+
+
```

Comment:
How about adding comment about __pycache__ like other parts of the file? And in this file, it tends to have a space between two sections.

e.g.

# Python cache files

Reply:
Done